contracts/interfaces/IAxelarGasService.sol

```diff
@@ -84,6 +84,8 @@ interface IAxelarGasService is IUpgradable {
 
     event NativeExpressGasAdded(bytes32 indexed txHash, uint256 indexed logIndex, uint256 gasFeeAmount, address refundAddress);
 
+    event Refunded(bytes32 indexed txHash, uint256 indexed logIndex, address payable receiver, address token, uint256 amount);
```

Comment:
```suggestion
    event Refunded(bytes32 indexed txHash, uint256 indexed logIndex, address payable indexed receiver, address token, uint256 amount);
```
Keep the receiver as indexed too, so we can query refunds for specific users easily?

Reply:
Kane didn't say it's needed. Adding it would make it not able to search by `txHash` and `logIndex` pair alone.

Reply:
Ah, that's a good point